src/post.ts

```diff
@@ -62,10 +62,15 @@ function numberology(): void {
   }
 }
 
+function updateMallPrices(): void {
```

Comment:
Possibly silly question: why do we want to update the snapshot every turn? why not just as we log out?

Reply:
or uhh stop running garbo. you know what i mean.

Reply:
This is to spread out the searches on mall prices for the items we collect. If we didn't do this and Garbo errors, the error would be hidden by all the messages of searching through the mall